src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The Bulldozer is doing some work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The Bulldozer stopped working");
```

Comment:
Similarly, the stopWork method message 'The Bulldozer stopped working' could be more descriptive. It would be beneficial to indicate the reason for the Bulldozer stopping its work or the state it is in after stopping.